### ISSUE TITLE
Accessibility issue in testing peek view

### ISSUE THREAD

**Issue by jdneo** [microsoft/vscode#145298]
The original issue comes from https://github.com/microsoft/vscode-java-test/issues/1387

Comment from @sevapopov2:

> Hello, here are more observations I have just made. It is possible to invoke the peek output view by shortcut, it's CTRL+; and m. Each time I invoke it, NVDA starts reading the test output out loud, but the focus is always placed at the line in the test file and as far as I understand it's selected. You can check that it's placed each time by closing the opened test file and invoking the output command again. Screen reader focus stays in the editor with a possibly selected code. I haven't found a way to browse the results by reading them slowly, line by line yet and be able to access links that are mentioned in the report if there are any. Thank you!

@connor4312 Looks like there is some problem with the editor focus when the test message view is peeked out?

**Comment by sevapopov2** [comment]
I found a way to view test results and work with tests.
When the tests are executed, I need to press CTRL+m for tab key to move between elements, then tab until the close button and after that there will be a tree with test results and different buttons to access such as go to the previous, next failures and others. That's accessible with NVDA, I am sure it's also accessible with Orca. Thank you for help all!

**Comment by connor4312** [comment]
Thanks for the issue!

@isidorn at the moment we reveal the failed position and then focus the editor. Instead is it preferable to focus within the peek view like the references peeks do?

https://github.com/microsoft/vscode/blob/c1d916181410459a82e9b66f8e7fd3b67e2138e2/src/vs/workbench/contrib/testing/browser/testingOutputPeek.ts#L751-L753

**Comment by isidorn** [comment]
@connor4312 yeah, it would be better if you focus inside the peek automatically for accessibility reasons.

If you want, you can also differentiate if the command is triggered via keybinding, if yes then move focus, otherwise do not. This is what the editor hover does.
Though I think it might be a bit confusing in this case, so I suggest to just pass focus inside the peek.

**Comment by sevapopov2** [comment]
Thank you for such an active feedback! I also think that it would be more convenient if the focus moves directly to the peek instead of moving it to the editor with the selected line that can be easily deleted by accedent. Thank you!

**Comment by VSCodeTriageBot** [comment]
This bug has been fixed in the latest release of [VS Code Insiders](https://code.visualstudio.com/insiders/)!

@jdneo, you can help us out by commenting `/verified` if things are now working as expected.

If things still don't seem right, please ensure you're on version 15253f1528573fc8f84e956b21af4bad57f38c07 of Insiders (today's or later - you can use `Help: About` in the command palette to check), and leave a comment letting us know what isn't working as expected.

Happy Coding!

**Comment by TylerLeonhardt** [comment]
@connor4312 how about some verification steps unless @sevapopov2 resurfaces?

**Comment by connor4312** [comment]
Steps in the original issue should still be accurate. Opening test peek output (for a failed test) should focus and screen-read the peek view, rather than staying in the editor.
